File: Architecture.Web/ClientApp/src/app/Shared/Entity/Users/QuestionInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class QuestionInfo extends Auditable {
    public QuestionInfoId: number;

    public profileId: number;
    
    public questionDescription: string;

    public pageToUrl: string;
   
    public sectionName: string;

    public sectionNameId: number;

    public status: any;
    
}

File: Architecture.Core/Entities/QuestionInfo.cs
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
    public class QuestionInfo : Auditable
    {
        public int QuestionInfoId { get; set; }
        public string QuestionDescription { get; set; }
        public string PageToUrl { get; set; }
        public int? SectionNameId { get; set; }
        public virtual SectionName SectionName { get; set; }
        public string Status { get; set; } 

    }
}
